samples/Nancy.Swagger.Annotations.Demo/Startup.cs

```diff
@@ -29,7 +31,33 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerF
                 app.UseDeveloperExceptionPage();
             }
 
-            app.UseOwin(x => x.UseNancy());
```

Comment:
Why don't you just add this to the Bootstrapper class that already exists? And just put some comments around the function you added to explain the purpose. 

When you ran this, did it create a swagger.json file in the repo? If so, you should make sure it's ignored via .gitignore.